content/en/cloud_cost_management/tag_pipelines.md

```diff
@@ -47,9 +47,15 @@ Use [Reference Tables][2] to add multiple tags to cost data without creating mul
 For example, if you want to add information about which VPs, organizations, and business_units different AWS and Azure accounts fall under, you can create a table and map the tags. Similar to Alias tag keys, the rule stops executing for each resource after a first match is found. For example, if an `aws_member_account_id` is found, then the rule no longer attempts to find a `subscriptionid`.
 {{< img src="cloud_cost/tags_mapmultiple.png" alt="Add account metadata like vp, organization, and businessunit using reference tables for tag pipelines" >}}
 
+## Reserved tags
+You can not add these tags in Tag Pipelines: `service`, `env`, `version`, and `host` becuase these tags are part of [Unified service Tagging][3]. 
+
+Since Tag Pipelines only apply to cost data, but these tags are improtant to helping correlate metrics, traces, and logs, we recommend you tag your underlying infrastructure with these tags so that you can take advantage of Unified Service Tagging across Datadog.
```

Comment:
Consider rephrasing/ breaking up into two sentences. Tag pipelines only apply to cost data. Reserved tags are more broadly applicable and help correlate metrics, traces and logs. So, we recommend that you tag your underlying infrastructure appropriately so you can take advantage of Unified Service Tagging across Datadog.

Reply:
thanks. i modified before i saw this comment, what do you think of my update?